doc/developer-notes.md

```diff
@@ -392,6 +392,8 @@ ln -s /path/to/project/root/src src
 
 3. Use `debugedit` to modify debug information in the binary.
 
+4. Change your breakpoints to use the file name only.
```

Comment:
Maybe add something like "if the IDE has an option for this" to make it slightly more clear, out of context, that it's an IDE option, and not something to set with say, `debugedit`.